ROADMAP.md

```diff
@@ -0,0 +1,62 @@
+# Parsec Project Roadmap
+
+## Notes
+
+- This roadmap document is a community resource. It is intended to help the project maintenance team
+   to share and evolve a common understanding of the project roadmap with all contributors and
+   stakeholders. It is for information purposes only, and it should be viewed as being both flexible
+   and approximate.
+- The release process for the Parsec project is documented
+   [here](https://parallaxsecond.github.io/parsec-book/contributing/release_process.html).
+- This roadmap document is focused on major/headline features and fixes. For full details and
+   changelogs for historical releases, please refer to the [releases
+   page](https://github.com/parallaxsecond/parsec/releases).
+- Full details of the current cryptographic capabilities of Parsec on the various platforms are
+   available
+   [here](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/service_api_coverage.html).
+- The section on previous releases does not include any that were made before there was a defined
+   process and cadence for Parsec releases.
+
+# Future Releases
+
+## 1.0 (March 2022)
+
+This release is being prepared. You can track the status
+[here](https://github.com/parallaxsecond/parsec/milestone/3).
+
+- Supports storage of key metadata using [SQLite](https://www.sqlite.org/index.html).
+- New [`CanDoCrypto`
+   API](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/can_do_crypto.html)
+   for runtime queries of cryptographic capabilities. This API is currently experimental and subject
+   to change.
+- New [APIs](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/attest_key.html)
+   in support of key attestation flows. These APIs are currently experimental and subject to change.
+   Currently supported for TPM platforms only.
+- [Java client library](https://github.com/parallaxsecond/parsec-client-java) introduced, including
+   integration with the Java Cryptography Architecture (JCA).
+- Parsec CLI tool supports the creation of Certificate Signing Requests (CSRs).
+- Integrates with [Trusted Services](https://www.trustedfirmware.org/projects/trusted-services/)
+   from [trustedfirmware.org](https://www.trustedfirmware.org) on platforms where these services are
+   available and functioning.
+
+## 1.x (September 2022)
+
+- Version number and content being defined.
+
+## Future Items (Not Assigned To Releases)
```

Comment:
and this before 1.x

Reply:
I'd prefer to have the futures section projecting into the future, similar to how a scrum/JIRA backlog would be structured, so I would tend to keep its current top-to-bottom structure. I don't think you should have to scroll down to find out what is coming imminently. And I don't think the roadmap should begin with a bucket of hypothetical items that are not yet assigned or committed to any projected release. Those things belong at the bottom, in my view.

There is yet a decision to be taken on what "direction" we choose for the Previous Releases section. For that one, it might make more sense to have the newest at the top. This would arguably be inconsistent, unless you take the view that the most relevant items are those that are closest to the present moment, and therefore belong at the top. But this question would be academic if we decide to lose this section altogether and just link to the releases page instead.